Classes/Private/Manager/Audio/OCTAudioEngine.m

```diff
@@ -54,8 +59,10 @@ @interface OCTAudioEngine ()
 @property (nonatomic, assign) AUNode ioNode;
 @property (nonatomic, assign) AudioUnit ioUnit;
 @property (nonatomic, assign) AudioComponentDescription ioUnitDescription;
-@property (nonatomic, assign) TPCircularBuffer buffer;
+@property (nonatomic, assign) TPCircularBuffer outputBuffer;
+@property (nonatomic, assign) TPCircularBuffer inputBuffer;
 @property (nonatomic, assign) OCTToxAVSampleRate currentAudioSampleRate;
+@property (nonatomic, assign) OCTToxAVSampleRate playbackSampleRate;
```

Comment:
Can we change rates to  `outputSampleRate` and `inputSampleRate` to be consistent?